### ISSUE TITLE
Can't load CSS from node_modules in NextJS

### ISSUE THREAD

**Issue by M1ck0** [zeit/next.js#10681]
# Bug report

## Describe the bug

I am trying to load CSS for npm library from node_modules but it fails every time 

## To Reproduce

package.json

```javascript
{
  "name": "template",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
    "dev": "next",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "@brainhubeu/react-carousel": "^1.12.0",
    "@zeit/next-sass": "^1.0.1",
    "bootstrap": "^4.4.1",
    "next": "^9.2.3-canary.12",
    "next-compose-plugins": "^2.2.0",
    "next-fonts": "^1.0.3",
    "node-sass": "^4.13.1",
    "prettier": "^1.19.1",
    "react": "^16.12.0",
    "react-bootstrap": "^1.0.0-beta.16",
    "react-carousel": "^4.3.0",
    "react-dom": "^16.12.0"
  }
}
```

/pages/_app.js
```javascript
import Head from "next/head"

import '@brainhubeu/react-carousel/lib/style.css'; //this .css file is a problem

import "../global.scss"

function MyApp({ Component, pageProps }) {
  return (
    <>
      <Head>
        <title>Starter App</title>
      </Head>
      <Component {...pageProps} />
    </>
  )
}

export default MyApp
```

next.config.js
```javascript

const withPlugins = require('next-compose-plugins')

const withSass = require('@zeit/next-sass')

const withFonts = require('next-fonts')

module.exports = withPlugins([withSass], [withFonts])
```

## Expected behavior

It should load web page normally and display its content

## System information

- OS: Manjaro Linux
- Browser: Mozilla Firefox, Google Chrome, Brave
- Version of Next.js: 9.2.3-canary.12

## Additional context

This is log from `yarn dev`
```javascript
yarn run v1.22.0
$ next
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
[ error ] ./node_modules/@brainhubeu/react-carousel/lib/style.css 1:0
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
```

**Comment by timneutkens** [comment]
Using `@zeit/next-sass` will disable built-in CSS support, can you try upgrading to `next@canary` and removing your `next.config.js` (as everything there is built-in on canary)

**Comment by M1ck0** [comment]
Well.. This is good to know. Thank you very much for your help.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
